SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialLoadBalanceEnforcer.h

```diff
@@ -3,52 +3,71 @@
 #pragma once
 
 #include "Interop/SpatialStaticComponentView.h"
+#include "Schema/AuthorityIntent.h"
+#include "Schema/ComponentPresence.h"
+#include "Schema/NetOwningClientWorker.h"
+#include "Schema/StandardLibrary.h"
 #include "SpatialCommonTypes.h"
 
-#include <WorkerSDK/improbable/c_worker.h>
-
-#include "CoreMinimal.h"
+#include "SpatialView/EntityComponentTypes.h"
+#include "SpatialView/EntityDelta.h"
```

Comment:
EntityDelta.h needed?

Reply:
nope- and nor was entity view! both gone